tests/setupconfig/test_auth.py

```diff
@@ -0,0 +1,185 @@
+from django.core.management import CommandError, call_command
+
+import pytest
+import requests
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from mozilla_django_oidc_db.models import (
+    OpenIDConnectConfig,
+    UserInformationClaimsSources,
+)
+from mozilla_django_oidc_db.setupconfig.bootstrap import AdminOIDCConfigurationStep
+
+from ..conftest import KEYCLOAK_BASE_URL
+
+
+@pytest.mark.django_db
+def test_configure(setup_config_full):
+    call_command("setup_configuration")
+
+    config = OpenIDConnectConfig.get_solo()
```

Comment:
since you're making queries and possibly hitting the django-solo cache, it would be good to set up an autouse fixture that clears the solo cache:

```py
@pytest.fixture(autouse=True)
def clear_solo_cache():
    yield
    OpenIDConnectConfig.clear_cache()